### DISCUSSION TITLE
青魔的6.45任務

### DISCUSSION THREAD

**Discussion by BonBM** [GpointChen/FFXIVChnTextPatch-GP#367] [General]
小的以日文原檔翻譯了青魔6.45的任務
CSV已經發在巴哈了
不過想問一下 如果用GP大大的程式 
我可以怎麼做才能只把quest/047的檔案丟進去FF?
因為試了幾次發現不能只丟quest檔 ,log會No resource files detected!
好像要連什麼東西一起丟 但我已經沒力研究了....

**Comment by GpointChen** [comment]
Achievement.csv

目前這個是寫死的，必須要讀到這個檔案，之後有空再修吧。
